docs/guides/src/main/server/db.adoc

```diff
@@ -30,35 +26,40 @@ The list of supported databases and their corresponding versions are:
 By default, the server uses the `dev-file` database. This is the default database that the server will use to persist data and
 only exists for development use-cases. The `dev-file` database is not suitable for production use-cases, and must be replaced before deploying to production.
 
-== Configuring a database
+By default, the Keycloak server uses the `dev-file` database to persist data. This database is intended for development environments only. It is unsuitable for a production environment and must be replaced before deploying to production.
```

Comment:
Should this substitute the paragraph before? If so, please delete the old paragraph, now it seems kind of doubled.

Reply:
(the untouched white paragraph and the new green one i mean)
![Screenshot 2022-02-10 at 12 18 39](https://user-images.githubusercontent.com/89905860/153396798-5adb4074-1a53-4e88-941b-cb4d53ffaced.png)